src/common/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { CSSProperties } from 'react'
+import { Range } from 'react-range'
+
+const SliderShape = ({
+  className,
+  style,
+}: {
+  className?: string
+  style?: React.CSSProperties
+}) => {
+  return (
+    <svg
+      className={className}
+      style={style}
+      viewBox='0 0 311 16'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+    >
+      <path
+        d='M0 8C0 6.88924 0.889079 5.98285 1.99963 5.96142L303.001 0.154312C307.394 0.0695737 311 3.60691 311 8C311 12.3931 307.394 15.9304 303.001 15.8457L1.99963 10.0386C0.889079 10.0172 0 9.11076 0 8Z'
+        fill='currentColor'
+      />
+    </svg>
+  )
+}
+
+export type SliderProps = {
+  min: number
+  max: number
+  style?: CSSProperties
+  formatValue?(value: number): number | string
+  formatTooltipValue?(value: number): number | string
+} & (
+  | {
+      value: number
+      onChange(value: number): void
+    }
+  | {
+      values: [number, number]
+      onChange(values: [number, number]): void
+    }
+)
+
+const defaultStyle = {
+  width: '311px',
```

Comment:
Can we use `rem` instead of `px`?

Reply:
[fixed](https://github.com/pastelnetwork/pastel-electron-wallet/pull/96/commits/2724487db1d22296a030fd92f8af10322aa67739)